firefox/package.json

```diff
@@ -6,7 +6,7 @@
 	"author": "honestbleeps",
 	"homepage": "http://redditenhancementsuite.com/",
 	"license": "GPL-3.0",
-	"version": "{{prop?version!../package.json}}",
+	"version": "{{to-string!./getVersion}}",
```

Comment:
Interesting. But amusingly, that page uses `0.1` in its example manifest, the "invalid semver" error message printed by jpm suggests `1.0.0rc4`, and AMO doesn't even let you submit extensions with a proper semver beta version:
![image](https://cloud.githubusercontent.com/assets/7673145/17148643/f9932ce4-5335-11e6-8953-984c8edd3bab.png)
Make up your mind Mozilla!